docs/blog/2018-06-08-life-after-layouts/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Life After Layouts
+date: 2018-06-08
+author: "Jason Lengstorf"
+---
+
+So the Gatsby V2 beta is out, and [layouts are going away](https://github.com/gatsbyjs/rfcs/blob/master/text/0002-remove-special-layout-components.md). What does this mean for your projects?
+
+This article will dive into the Gatsby V2's approach to layouts, talk about what changed and why, and walk through the migration process to get your existing projects updated to V2.
+
+## What changed?
+
+In Gatsby V1, layouts were a special kind of component that automagically wrapped generated templates. They were stored in `src/layouts/`, and received a prop called `children` that needed to be executed as a function.
```

Comment:
we generally say "page components" instead of templates.

Reply:
Updated!